vault-backed/aws/aws.tf

```diff
@@ -92,11 +96,14 @@ resource "aws_iam_user_policy" "vault_secrets_engine_generate_credentials" {
     Statement = [
       {
         Action = [
-          "sts:AssumeRole",
+          "iam:GetUser",
+          "iam:CreateAccessKey",
+          "iam:DeleteAccessKey",
+          "iam:ListAccessKeys"
```

Comment:
Given that this policy uses the `assumed_role` credential_type, I'm confused why this policy would need to be able to deal with access keys 🤔

Reply:
Sorry! It's been a long time since we looked at this. I see now what's happening. This is totally independent from the credential_type 👍